layouts/partials/developer-hub/application.html

```diff
@@ -0,0 +1,51 @@
+{{$servicesData := site.Data.developerhub.services }}
+{{$services := cond (ne (len .services) 0)  (printf " service-%s" (delimit .services " service-")) "" }}
+{{$deployment := cond (ne (len .deployment) 0)  (printf " deployment-%s" (delimit .deployment " deployment-")) "" }}
+{{$platform := cond (ne (len .platform) 0)  (printf " platform-%s" (delimit .platform " platform-")) "" }}
+{{$level := cond (ne (len .complexity) 0)  (printf " level-%s" (delimit .complexity " level-")) "" }}
+{{$cloudpod := cond (eq (.cloudPods) true) " cloudpod-true" "" }}
+{{$className := print $services $deployment $platform $level $cloudpod}}
+<div
+    class='entry application-card filters-card col-xs-12 col-sm-12 col-md-4{{$className}}'
```

Comment:
This should be changed to
```
    class='entry application-card filters-card col-xs-12 col-sm-6 col-lg-4{{$className}}'
```

That way it's consistent with the width of tutorial cards across all screen sizes.

Otherwise it looks like this on smaller screens:

<img width="793" alt="Screenshot 2023-09-12 at 12 21 26" src="https://github.com/localstack/docs/assets/39307517/b947ce29-081d-4b0b-ad30-ce8de5840249">

Reply:
awesome, thanks for the suggestion!